decred/tests/unit/crypto/test_rando.py

```diff
@@ -0,0 +1,14 @@
+"""
+Copyright (c) 2020, The Decred developers
+See LICENSE for details
+"""
+
+import pytest
+
+from decred import DecredError
+from decred.crypto import rando
+
+
+def test_checkSeed():
+    with pytest.raises(DecredError):
+        rando.checkSeed(1)
```

Comment:
I didn't add them because the rest of the code was already covered by other tests, but it does make sense to have decently comprehensive specific tests.